content/management/configuration-api/v3.2/index.mdx

```diff
@@ -32,6 +32,720 @@ Summing up, if the user token starts with `fra-`, you should add the `X-Region:
+
+#### Request
+
+| Parameter                            | Required | Data type | Notes                       |
+| ------------------------------------ | -------- | --------- | --------------------------- |
+| `id`                                 | Yes      | `string`  | Agent ID                    |
+
+#### Response
+
+No response payload (`200 OK`).
+
+</Text>
+
+<Code>
+
+<CodeSample path={'REQUEST'}>
+
+```shell
+curl -X POST \
+  https://api.livechatinc.com/v3.2/configuration/action/suspend_agent \
+  -H 'Authorization: Bearer <your_access_token>' \
+  -H 'Content-Type: application/json' \
+  -d '{
+        "id": "agent1@domain.com"
+      }'
+```
+
+</CodeSample>
+
+</Code>
+
+</Section>
+
+<Section>
+
+<Text>
+
+### Unsuspend Agent
+
+Unsuspends Agent specified by `id`.
+
+#### Specifics
+
+|                 |                                                                         |
+|-----------------|-------------------------------------------------------------------------|
+| Method URL      | `https://api.livechatinc.com/v3.2/configuration/action/unsuspend_agent` |
+| Required scopes | `agents--all:rw` `billing:rw`                                           |
+
+#### Request
+
+| Parameter                            | Required | Data type | Notes                       |
+| ------------------------------------ | -------- | --------- | --------------------------- |
+| `id`                                 | Yes      | `string`  | Agent ID                    |
+
+#### Response
+
+No response payload (`200 OK`).
+
+</Text>
+
+<Code>
+
+<CodeSample path={'REQUEST'}>
+
+```shell
+curl -X POST \
+  https://api.livechatinc.com/v3.2/configuration/action/unsuspend_agent \
+  -H 'Authorization: Bearer <your_access_token>' \
+  -H 'Content-Type: application/json' \
+  -d '{
+        "id": "agent1@domain.com"
+      }'
+```
+
+</CodeSample>
+
+</Code>
+
+</Section>
+
+<Section>
+
+<Text>
+
+### Request Agent Unsuspension
+
+Suspended agent can send email to all owners and viceowners with request to unsuspend him.
+
+#### Specifics
+
+|                 |                                                                                    |
+|-----------------|------------------------------------------------------------------------------------|
+| Method URL      | `https://api.livechatinc.com/v3.2/configuration/action/request_agent_unsuspension` |
+| Required scopes |                                                                                    |
+
+#### Request
+
+Empty request payload.
+
+#### Response
+
+No response payload (`200 OK`).
+
+</Text>
+
+<Code>
+
+<CodeSample path={'REQUEST'}>
+
+```shell
+curl -X POST \
+  https://api.livechatinc.com/v3.2/configuration/action/request_agent_unsuspension \
+  -H 'Authorization: Bearer <your_access_token>' \
+  -H 'Content-Type: application/json' \
+  -d '{}'
+```
+
+</CodeSample>
+
+</Code>
+
+</Section>
+
+<Section>
+
+<Text>
+
+### Approve Agent
+
+Approves agent making him able to use application.
```

Comment:
```suggestion
Approves an Agent thus allowing the Agent to use the application.
```

Reply:
We should avoid saying "him" or "her", and be more generic. It's possible to say "them", but in this example, it would sound weird.